site/src/components/address-input/zipGeocoding.ts

```diff
@@ -0,0 +1,46 @@
+import { StateCode, USZipCode } from "./SmartyStreets";
+import { ZipGeocoding } from "./zipGeocoding.d";
+
+const smartyApiURL = "https://us-zipcode.api.smartystreets.com/lookup";
+
+export default async (zipcode: string, authId: string): Promise<ZipGeocoding.Result | ZipGeocoding.Error> => {
```

Comment:
You can just include the two interfaces in this file since files are their own modules. Just makes things a bit cleaner and easier to read.

Reply:
Also just an FYI, but you can use `.ts` for all files. The `.d.ts` extension is really only for referencing JS code. If a `.ts` file only has types and no usable code the compiler will just elide it when producing the js output.

Reply:
@groeneman Also just an FYI, but you can use .ts for all files. The .d.ts extension is really only for referencing JS code. If a .ts file only has types and no usable code the compiler will just elide it when producing the js output.